packages/core/parcel-bundler/src/assets/CSSAsset.js

```diff
@@ -125,11 +149,40 @@ class CSSAsset extends Asset {
         'module.exports = ' + JSON.stringify(this.cssModules, null, 2) + ';';
     }
 
+    if (this.options.sourceMaps) {
+      if (this.sourceMap) {
+        this.sourceMap = await new SourceMap().addMap(this.sourceMap);
+      }
+
+      if (this.previousSourceMap) {
+        this.previousSourceMap.sources = this.previousSourceMap.sources.map(v =>
+          path.join(
+            path.dirname(this.relativeName),
+            this.previousSourceMap.sourceRoot || '',
+            v
+          )
+        );
+        if (this.sourceMap) {
+          this.sourceMap = await new SourceMap().extendSourceMap(
+            this.previousSourceMap,
+            this.sourceMap
+          );
+        } else {
+          this.sourceMap = await new SourceMap().addMap(this.previousSourceMap);
+        }
+      } else if (!this.sourceMap) {
+        this.sourceMap = new SourceMap().generateEmptyMap(
+          this.relativeName,
+          css
+        );
+      }
+    }
+
     return [
       {
         type: 'css',
         value: css,
-        cssModules: this.cssModules
```

Comment:
In VueAsset, it's called `modules: `. Should this be `modules: ` as well or isn't it used at all?